Libplanet/Net/Messages/GetBlocks.cs

```diff
@@ -0,0 +1,58 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography;
+using NetMQ;
+
+namespace Libplanet.Net.Messages
+{
+    internal class GetBlocks : Message
+    {
+        public GetBlocks(
+            BlockLocator locator, HashDigest<SHA256>? stop)
+        {
+            Locator = locator;
+            Stop = stop;
+        }
+
+        public BlockLocator Locator { get; }
+
+        public HashDigest<SHA256>? Stop { get; }
+
+        protected override MessageType Type => MessageType.GetBlocks;
+
+        protected override IEnumerable<NetMQFrame> DataFrames
+        {
+            get
+            {
+                yield return new NetMQFrame(
+                    NetworkOrderBitsConverter.GetBytes(Locator.Count()), 4);
```

Comment:
What is the reason for limiting the length of the buffer?

Reply:
There isn't serious reason, `Locator.Count()` is `int` and the size of `int` is 4 (bytes).

Reply:
Now that I think about it, it's fine to remove buffer size. I'll remove it.